src/main/java/uk/gov/companieshouse/insolvency/delta/consumer/InsolvencyDeltaConsumer.java

```diff
@@ -26,21 +26,23 @@ public InsolvencyDeltaConsumer(InsolvencyDeltaProcessor deltaProcessor) {
     /**
      * Receives Main topic messages.
      */
-    @KafkaListener(topics = "${insolvency.delta.topic.main}", groupId = "insolvency.delta.topic.main")
+    @KafkaListener(topics = "${insolvency.delta.topic.main}", groupId = "insolvency-delta-consumer")
```

Comment:
Why hardcode the groupId? I think it should be a property e.g. ${insolvency.delta.groupid} or defined in consumer config?